schemas/cs3/tbl/t_item/item.ksy

```diff
@@ -0,0 +1,109 @@
+meta:
+  id: item
+  endian: le
+  encoding: UTF-8
+  imports:
+    - /components/cs3/character
+    - /components/cs3/item_type
+    - /components/cs3/effect
+    - /components/common/element
+seq:
+  - id: item_id
+    type: u2
+  - id: character_restriction
+    type: character
+  - id: flags
+    type: strz
+  - id: item_type
+    type: item_type
+  - id: item_subtype
+    type: u2 # TODO: make an enum for this?
+
+  - id: element
+    type: u1 # can't use element enum here because it's a u2
+
+  - id: slash
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: thrust
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: pierce
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: strike
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: unknown_byte
+    type: u1
+    doc: |
+      Only nonzero for (some) weapons.
+  - id: target_type
+    type: u1
+    doc: |
+      For weapons and consumables, this is the target type.
+      However, this field is also nonzero for a few other items (such as Eternal Kandelaar and costumes)
```

Comment:
Do those have any special functions that we can think of?

Reply:
It seems to be the type for costumes (e.g. 1 for hats), some sort of id for arcus covers (they're numbered 1 through 11), and I don't know what it does for quest items like Kandelaar (though since it seems to be 10 always my guess would be it does nothing)